src/main/Member.js

```diff
@@ -199,6 +199,19 @@ export default class Member {
     }
 
     /**
+     * Set the "normal consumer" rule as member's recovery rule.
+     * (As of Nov 2017, this rule was: To recover, verify an alias.)
+     * @return {Promise} promise containing RecoveryRule proto buffer.
+     */
+    useDefaultRecoveryRule() {
+        return Util.callAsync(this.useDefaultRecoveryRule, async () => {
+            const prevHash = await this._getPreviousHash();
+            const res = await this._client.useDefaultRecoveryRule(prevHash);
```

Comment:
It looks a little weird that this function takes only preview hash. Do you think it is better that we construct the recoveryRule here and call something like addRecoveryRule in the AuthHttpClient, which looks more generic and consistent with calls like addAlias and addKeys